pom.xml

```diff
@@ -2,16 +2,30 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>juja.sqlcmd</groupId>
+    <groupId>juja</groupId>
     <artifactId>sqlcmd</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <build>
```

Comment:
both blocks do the same functionality but java versions inside of them are different